# doi:
http://dx.doi.org/10.1016/j.compbiomed.2021.104611

# Abstract

RNA-protein interactions of a virus play a major role in the replication of RNA viruses. The replication and transcription of these viruses take place in the cytoplasm of the host cell; hence, there is a probability for the host RNA-viral protein and viral RNA-host protein interactions. The current study applies a high-throughput computational approach, including feature extraction and machine learning methods, to predict the affinity of protein sequences of ten viruses to three categories of RNA sequences. These categories include RNAs involved in the protein-RNA complexes stored in the RCSB database, the human miRNAs deposited at the mirBase database, and the lncRNA deposited in the LNCipedia database. The results show that evolution not only tries to conserve key viral proteins involved in the replication and transcription but also prunes their interaction capability. These proteins with specific interactions do not perturb the host cell through undesired interactions. On the other hand, the hypermutation rate of NSP3 is related to its affinity to host cell RNAs. The Gene Ontology (GO) analysis of the miRNA with affiliation to NSP3 suggests that these miRNAs show strongly significantly enriched GO terms related to the known symptoms of COVID-19. Docking and MD simulation study of the obtained miRNA through high-throughput analysis suggest a non-coding RNA (an RNA antitoxin, ToxI) as a natural aptamer drug candidate for NSP5 inhibition. Finally, a significant interplay of the host RNA-viral protein in the host cell can disrupt the host cell's system by influencing the RNA-dependent processes of the host cells, such as a differential expression in RNA. Furthermore, our results are useful to identify the side effects of mRNA-based vaccines, many of which are caused by the off-label interactions with the human lncRNAs.


# Highlights
* The role of physical interaction of viral protein-host miRNA in the pathogenicities of the coronavirus infection.
* High Throughput Analysis of Viral Protein Interactions with Host Cell RNAs.
* ToxI (antitoxin miRNA in the Toxinâ€“antitoxin system) can inhibit NSP5 of CoVID-19.
* NSP3 hyper-mutation rate has an effective role in the evolution of covid-19.
* NSP3 hyper-affinity to RNAs has an effective role in covid-19 pathogenicity.

# High-Throughput Analysis of the Interactions between Viral Proteins and Host Cell RNAs 
* Supplementary File 1: Frequency of RNAs binding to the proteins of SARS-CoV-2
* Supplementary File 2: Prediction score matrix of RNAs involved in complexes with proteins of SARS-CoV-2
* Supplementary File 3: Prediction score matrix of miRNAs with proteins of SARS-CoV-2
* Supplementary File 4: Prediction score matrix of lncRNAs with proteins of SARS-CoV-2
* Supplementary File 5: All significant gene ontology details of miRNAs target
* Supplementary File 6: The Sequences of the proteins of 10 different viruses 
* Supplementary File 7: The rmsd plot of MD simulation
